### DISCUSSION TITLE
Suggestion: Add VueJS and other mentioned examples and general support

### DISCUSSION THREAD

**Discussion by aaronlippold** [markdoc/markdoc#420] [Ideas]
The docs mention VueJS and other languages as usable platforms but no examples are provided.

<https://markdoc.dev/docs/render#create-your-own>

If possible, examples for these other mentioned use cases could help general adoption.

My team for examples uses VueJS and Nuxt and Tailwind but are looking at MarkDoc as a possible solution to making MD <-> XML and MD <-> JSON documentation systems.

**Comment by mfix-stripe** [comment]
@aaronlippold here is an example of using Markdoc w/ Vue: https://github.com/markdoc/markdoc/discussions/25

I haven't seen an example shared using Markdoc w/ Nuxt, but if you build it, please share it in our [Show and Tell](https://github.com/markdoc/markdoc/discussions/categories/show-and-tell)!
